README.md

```diff
@@ -122,11 +128,26 @@ pieces create
 
 ##### Edit an existing asset:
 
-This currently only works for an assets's name
+This will edit the name and reclassify the current asset.
```

Comment:
I still have to try this out but I am unsure about the paradigm if the 'current' asset. This might need to live as its own issue but how can I set the 'current' asset, also is there a good way to check what the current asset is?

Reply:
You can set the current asset by using the open command this is jayson idea and I like it but if u have a better way in mind lemme know